app/javascript/src/infinite-scroll.js

```diff
@@ -64,6 +64,20 @@ function getInfiniteScrollContent(element) {
         element.innerHTML = "Load more"
         element.setAttribute("data-url", nextUrl)
       }
+    },
+    error: (error) => {
+      progressBar.setValue(1)
+      progressBar.hide()
+
+      const spinner = document.querySelector(".items").querySelector(".spinner")
+      if (spinner) spinner.remove()
+      if (element.classList.contains("button")) {
```

Comment:
Not the nicest way of checking this, if we change the style of the button (and with that the class name) this will fail, but meh, that's a problem for another day

Reply:
Shouldn't be too much work to add an extra `data-*` attribute to distinguish the two elements. I'll take care of that now to avoid technical debt.